app/lib/actions/katello/content_view/publish.rb

```diff
@@ -4,6 +4,7 @@ module ContentView
       class Publish < Actions::EntryAction
         middleware.use Actions::Middleware::KeepCurrentUser
 
+        # rubocop:disable MethodLength
```

Comment:
i disabled it because adding my one line caused the method to go over the limit and re-factoring it here seemed wrong.  I don't know that i like disabling it for all actions, but 5 actions already have it disabled on the plan method.